msticpy/common/wsconfig.py

```diff
@@ -92,7 +93,13 @@ def __init__(self, config_file: Optional[str] = None, workspace: str = None):
                     "\n".join(_NO_CONFIG_WARN).format(config_file=config_file)
                 )
         self._config_file = config_file
-        self._config.update(self._read_config_values(config_file))
+        config = self._read_config_values(config_file)
+        if config is not None:
```

Comment:
Not sure about this. config_file is meant to be a config.json. I guess the doc string is misleading - I originally meant for this to be a way to override msticpyconfig and force load of a json file. I guess if it works specifying msticpyconfig.yaml as config_file maybe we should keep it.